spec/coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 97.64
+    "covered_percent": 97.68
```

Comment:
remove all spec/coverage files from commit and do a force push. we don't want any of these files in the git repo.